src/pro/java/hw5/competition/Main.java

```diff
@@ -0,0 +1,66 @@
+package pro.java.hw5.competition;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Main {
+
+  private static final Random random = new Random();
+
+  public static void main(String[] args) {
+
+    /* max length = 200, max height = 4 */
+    Human petya = new Human("Петя", getRandomRunRange(), getRandomJumpRange());
+
+    /* max length = 300, max height = 5 */
+    Cat murka = new Cat("Мурка", getRandomRunRange(), getRandomJumpRange());
+
+    /* max length = 800, max height = 2 */
+    Robot bot = new Robot("МКБот", getRandomRunRange(), getRandomJumpRange());
+
+    Treadmill runOne = new Treadmill();
+    Wall wallOne = new Wall();
+
+    List<Player> players = new ArrayList<>();
+    List<Obstacle> obstacles = new ArrayList<>();
+
+    players.add(petya);
+    players.add(murka);
+    players.add(bot);
+
+    obstacles.add(runOne);
+    obstacles.add(wallOne);
+
+    for (Player player : players) {
+
+      if (player.run().contains("не")) {
```

Comment:
Try to avoid conditions as this. Firstly, please use english. Secondary, what if somebody changes returned value of player.run() method? It's too sensitive condition, for example you can use boolean type of method which can contain false or true

Reply:
![image](https://github.com/kajihan/java_practice/assets/141428968/4871cb1e-2ff3-4c1e-a630-b4a93a1e7788)